### ISSUE TITLE
data format, 

### ISSUE THREAD

**Issue by chocosando** [hmshan/CPCE-3D#3]
Hi. Thank you for nice talk!

How can I use this code for my image ? 

My images consist of :
   pair of  routine radiation dose CT [ 512 x 512] and low dose CT [ 512 x 512 ] 

How can I make N x D x W x H  and N x W x H ? 
What does 'label' means for paired images ?

Thank you!!

**Comment by hmshan** [comment]
Hi there,

The label here means the routine radiation dose CT image. The code here has a 2D version and 3D version. Since the training process is usually based on image patches, so the W and H are 64 in the paper. 

1. 2D version. The training data and label can both be N x 64 x 64, where N is the number of image patches.
2. 3D version. The 3D version requires the adjacent slices and 3D convolution. So the input needs to be N x D x 64 x 64, and the label corresponds to the central slice (D//2). For example, if D=3, every time you need to input three adjacent slices and the output of the network corresponds to the central slice. Put differently, the two adjacent slices of the central one will help to denoise the central slice. 

![image](https://user-images.githubusercontent.com/3102105/53380159-695b7a80-393a-11e9-9c24-4ded3461ac9e.png)
